client/js/views/Header.js

```diff
@@ -12,8 +12,8 @@ module.exports = Object.create( Object.assign( {}, require('./__proto__'), {
 
     name: 'Header',
     onNavListClick(event) {
-        this.emit('navigate', `/${ event.target.textContent.replace(/\s+/g, '') }`)
-        var headers = [...event.target.parentElement.children].forEach(header => header.classList.remove('selected'))
+        this.emit('navigate', `/${ event.target.textContent.replace(/\s+/g, '') }`);
+        [...event.target.parentElement.children].forEach(header => header.classList.remove('selected'))
         event.target.classList.add('selected')
```

Comment:
So continuing on what I said above, you can just go ``this.emit( 'navigate', `/${event.target.getAttribute('data-name')}` )`` and boom, you've got the right route without having to perform a replace.

Reply:
I actually like this solution too.  Keeping track of `selectedNav` isn't necessary.  Its simple, albeit slower, to remove all, then add one.